addons/gsri_opex/functions/core/fn_initSettings.sqf

```diff
@@ -6,10 +8,10 @@ _settings = (configProperties [configFile >> "GSRI_FREMM_Templates" >> "GSRI_Nor
 
 // Automatic creation of needed server-forced settings for
 {
-	_template = _x;
+	private _template = _x;
 	{
-		_settingType = ["CHECKBOX","EDITBOX"] select (isText (configFile >> "GSRI_FREMM_Templates" >> _template >> _x));
-		_defaultValue = switch (_settingType) do {
+		private _settingType = ["CHECKBOX","EDITBOX"] select (isText (configFile >> "GSRI_FREMM_Templates" >> _template >> _x));
+		private _defaultValue = switch (_settingType) do {
```

Comment:
Not sure how this works

Reply:
What are you unsure of ?

Reply:
It's just I don't understand what you want to achieve with this code.
Is the switch used to behave depending on the selected template ?

Reply:
We need to create different types of CBA settings following what is configured in the template's config :
- for custom textures, name, numbers, which are registered as text values, we need an editbox
- for features like weapons, selectable helicopter, bays, which are registered as numeric booleans, we need a checkbox.
See https://github.com/team-gsri/GSRI-Opex/blob/20de941b7467e42c1136d7841361f9ecbe38560a/addons/gsri_opex/config.cpp#L46 for the configuration of the template